hypervisor/arch/x86/vtd.c

```diff
@@ -471,7 +471,7 @@ static int32_t dmar_register_hrhd(struct dmar_drhd_rt *dmar_unit)
 
 	dmar_unit->cap = iommu_read64(dmar_unit, DMAR_CAP_REG);
 	dmar_unit->ecap = iommu_read64(dmar_unit, DMAR_ECAP_REG);
-	dmar_unit->gcmd = iommu_read32(dmar_unit, DMAR_GCMD_REG);
+	dmar_unit->gcmd = iommu_read32(dmar_unit, DMAR_GSTS_REG);
```

Comment:
Oh, somebody changed it wrongly...
Can u add some comments here to avoid future mistake?

Reply:
Sure. I just pushed a new version with comments in code.